components/Modal.styles.js

```diff
@@ -0,0 +1,40 @@
+import { StyleSheet } from 'react-native';
```

Comment:
the font sizes are looking slightly bigger than from the menu component.

Reply:
Yeah, before they looked so small. Will remove the sizing again and see if @mcastro-lqd complains. :p